contracts/extensions/votingReputation/VotingReputationStorage.sol

```diff
@@ -214,14 +213,11 @@ contract VotingReputationStorage is ColonyExtension, BasicMetaTransaction, Votin
       return MotionState.Finalized;
     } else if (_motionId <= motionCountV10 && getSig(motion.action) == MULTICALL) {
```

Comment:
Is there something stopping us making this

```
} else if (_motionId <= motionCountV10 && (getSig(motion.action) == NO_ACTION || getSig(motion.action) == OLD_MOVE_FUNDS)) {
   return MotionState.Finalized;
} else {
  return MotionState.Finalizable;
}
```

Reply:
Oh, I guess we want to guard against Multicall hiding a forbidden `OLD_MOVE_FUNDS` going forward, even if we can't do anything about it.